src/views/SystemList/index.tsx

```diff
@@ -25,29 +25,42 @@ import PageWrapper from 'components/PageWrapper';
 import Alert from 'components/shared/Alert';
 import { ErrorAlert, ErrorAlertMessage } from 'components/shared/ErrorAlert';
 import SectionWrapper from 'components/shared/SectionWrapper';
-import GetCedarSystemsQuery from 'queries/GetCedarSystemsQuery';
+import GetCedarSystemsAndBookmarksQuery from 'queries/GetCedarSystemsAndBookmarksQuery';
 import {
-  GetCedarSystems,
-  GetCedarSystems_cedarSystems as CedarSystem
-} from 'queries/types/GetCedarSystems';
-import { mockBookmarkInfo } from 'views/Sandbox/mockSystemData'; // TODO - replace mockSystemInfo/mockBookmarkInfo with dynamic data fetched from backend and CEDAR
+  GetCedarSystemsAndBookmarks,
+  GetCedarSystemsAndBookmarks_cedarSystemBookmarks as CedarSystemBookmark,
+  GetCedarSystemsAndBookmarks_cedarSystems as CedarSystem
+} from 'queries/types/GetCedarSystemsAndBookmarks';
 
 import Table from './Table';
-import { filterBookmarks } from './util';
+import filterBookmarks from './util';
 
 import './index.scss';
 
 export const SystemList = () => {
   const { t } = useTranslation('systemProfile');
 
   // TODO: query parameters and caching
-  const { loading, error, data } = useQuery<GetCedarSystems>(
-    GetCedarSystemsQuery
-  );
+  const {
+    loading,
+    error,
+    data,
+    refetch
+  } = useQuery<GetCedarSystemsAndBookmarks>(GetCedarSystemsAndBookmarksQuery);
+
+  const systemsTableData = (data?.cedarSystems
```

Comment:
We would be able to simplify this (as with the `bookmarks` definition below if our GraphQL schema was more strictly typed; as it is, `data?.cedarSystems ?? []` is of type `(CedarSystem | null)[]`.
No change needs to be made for this on this PR, but I created [EASI-1518](https://jiraent.cms.gov/browse/EASI-1518) for cleaning it up later.